README.md

```diff
@@ -76,10 +79,14 @@ yield message_chunk("Hello, world!")
 ```
 
 ### `reasoning_step`
+OpenBB Workspace allows you to return "reasoning steps" (sometimes referred to
+as "thought steps" or even "status updates") from your custom agent to the
+front-end, at any point in the agent's execution. This is often useful for
+providing updates and extra information to the user, particularly for
+long-running queries, or for complicated workflows.
 
-Create a reasoning step (also known as a status update) SSE to communicate the
-status of the agent, or any additional information as part of the agent's
-execution to OpenBB Workspace.
+To send a reasoning step SSE to OpenBB Workspace, use the `reasoning_step`
+helper function:
```

Comment:
As long as the reasoning_step helper returns a model the example below is incorrect